lib/p2p/NodeList.ts

```diff
@@ -1,17 +1,38 @@
 import P2PRepository from './P2PRepository';
 import { NodeInstance, NodeFactory } from '../types/db';
-import { Address } from '../types/p2p';
+import { Address, ReputationEvent } from '../types/p2p';
 import addressUtils from '../utils/addressUtils';
+import { ReputationEventType } from '../types/enums';
+import { ms } from '../utils/utils';
+import { EventEmitter } from 'events';
+
+const ReputationEventWeight = {
+  [ReputationEventType.ManualBan]: -100000000,
+  [ReputationEventType.PacketTimeout]: -5,
+  [ReputationEventType.SwapTimeout]: -15,
+  [ReputationEventType.SwapSuccess]: 1,
```

Comment:
I think eventually we'll want something like a `MANUAL_UNBAN` or `MANUAL_CONNECT` event for when we intentionally reconnect to a peer we have already banned. That should maybe be a special case where we reset the score to 0, rather than adding or subtracting. Just a thought.